src/data/columnAttributes.tsx

```diff
@@ -0,0 +1,35 @@
+const small = 200; //10rem;
```

Comment:
This should be merged with `FieldToViewMappings.tsx` and renamed to something like `ResultsColumns` or `ColumnConfiguration`?

Reply:
Nice idea. Done.